[
    {
        "type": "message",
        "user": "U3JKB7NM6",
        "text": "Hi all, For sawtooth, while generating keys, I encounter the following error",
        "ts": "1483014640.000200"
    },
    {
        "type": "message",
        "user": "U3JKB7NM6",
        "text": "Command\n.\/bin\/sawtooth keygen --key-dir validator\/keys mkt\n\nError\nfrom sawtooth_signing.ECDSA import ECDSARecoverModule as nativeECDSA\nImportError: cannot import name ECDSARecoverModule",
        "ts": "1483014676.000201"
    },
    {
        "type": "message",
        "user": "U3JKB7NM6",
        "text": "someone has some idea, pls",
        "ts": "1483014688.000202"
    },
    {
        "type": "message",
        "user": "U3JKB7NM6",
        "text": "I imported python's ecdsa module as well",
        "ts": "1483014705.000203"
    },
    {
        "type": "message",
        "user": "U1L6PRGTF",
        "text": "That seems like the \/bin\/build_all script hasn't been run. Did you run it?",
        "ts": "1483017104.000204"
    }
]